pontoon/sync/core/entities.py

```diff
@@ -333,6 +333,18 @@ def entities_same(a: Entity, b: Entity) -> bool:
     )
 
 
+def entity_update(
+    current: Entity, update_from: Entity, fields: list[str]
+) -> Optional[Entity]:
```

Comment:
This is replacing the only use of `entities_same()` with a new function; we should not leave orphaned code lying around.